src/sage/schemes/curves/constructor.py

```diff
@@ -76,8 +72,8 @@
                            IntegralAffinePlaneCurve,
                            IntegralAffinePlaneCurve_finite_field)
 
-
-from sage.schemes.plane_conics.constructor import Conic
+_Fields = Fields()
+_NumberFields = NumberFields()
```

Comment:
Same comment. I don't see the advantages of having yet another place where these are nailed in memory.

Reply:
Done.